docker-compose.yml

```diff
@@ -115,41 +149,57 @@ services:
       - ZENODO_ACCESS_TOKEN
       - ZOTERO_API_KEY
       - ZOTERO_LIBRARY
+    image: ${COMPOSE_PROJECT_NAME-rsd}/harvesting
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "1"
+        max-size: "80m"
     volumes:
       - ./docker-volumes/oaipmh-cache:/app/oaipmh-cache
       - static_schedule:/static_schedule
 
   nginx_ssl:
     container_name: ${COMPOSE_PROJECT_NAME-rsd}-nginx-ssl
-    image: ${COMPOSE_PROJECT_NAME-rsd}/nginx_ssl
+    depends_on:
+      - reverse-proxy
     environment:
       # user should not have to change the values of the following environment variables:
       - SSL_FORWARD=http://reverse-proxy:80
       # user needs to assign values to the following environment variables:
       - SSL_ADMIN_EMAIL
       - SSL_DOMAINS
+    image: ${COMPOSE_PROJECT_NAME-rsd}/nginx_ssl
```

Comment:
It should be only `image: rsdnlesc/docker-term-letsencrypt`. This image is from Dockerhub, not built using the RSD.

Speaking of this image, a nicer alternative could be this: https://docs.traefik.io/user-guides/docker-compose/basic-example/ . It works by using labels in the docker-compose file, and has a lot more features.

Reply:
remove second image, made an issue for potentially replacing the docker image